src/main/java/com/courtcanva/ccfranchise/models/Order.java

```diff
@@ -0,0 +1,89 @@
+package com.courtcanva.ccfranchise.models;
+
+
+import com.courtcanva.ccfranchise.constants.OrderStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.UpdateTimestamp;
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "`order`")
+// @TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
```

Comment:
if you are not using jsonb, have to make sure it's working in both H2 and postgre

Reply:
and also, this Order is a new class, I only saw the script to update status. please double check

Reply:
In database, table `order `already exists. It's created by `db/migration` `v1`